docs/installation.md

```diff
@@ -5,6 +5,12 @@ a Marionette application. In this section we'll cover the most common ways.
 
 
 ## Quick start using NPM and Webpack
+[NPM](https://www.npmjs.com/) is the package manager for JavaScript.
+
+Installing with NPM through command-line interface
+```
+npm install backbone.marionette@3.0.0
```

Comment:
1. `npm install backbone.marionette` is enough.
2. `npm install backbone.marionette --save` could also be good

I tested this on my machine, and 3.0.0 installed without forcing version install.
I want to guess that
> npm install backbone.marionette latest-version installing version 2.4.7 instead 3.0.0

might have happened because your package.json had ^2.4.x, which won't install 3.x version.

Anyhow, 1 or 2 might be a better alternative. I'll let other chime in regarding this message.

Thanks for the PR.